addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -327,10 +332,15 @@ var SlideUploadDialog = Dialog.extend({
      */
     _setup: function () {
         this.slide_category_data = {
-            presentation: {
+            document: {
                 icon: 'fa-file-pdf-o',
-                label: _t('Presentation'),
-                template: 'website.slide.upload.modal.presentation',
+                label: _t('Document'),
+                template: 'website.slide.upload.modal.document',
+            },
+            infographic: {
```

Comment:
image maybe, as we now use image ? Note that keeping infographic helps grepping it, thinking of it.

Reply:
This matches the slide_categories, so it's still "infographic" (the technical value).